pyproject.toml

```diff
@@ -29,6 +29,14 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
   "bioio-base>=0.1.1",
+  "dask[array]>=2021.4.1",
+  "fsspec>=2022.8.0",
+  "imageio[ffmpeg]>=2.11.0,<2.28.0",
+  "numpy>=1.16.0,<2.0.0",
+  "ome-types>=0.3.3",
```

Comment:
I think ome-types 0.3.4, but also see this https://forum.image.sc/t/ome-types-rewritten-requesting-alpha-testers/83174

Reply:
https://ome-types.readthedocs.io/en/latest/migration/

Reply:
while that migration guide looks like a lot, you should also know that I routinely [test against aicsimageio](https://github.com/tlambert03/ome-types/actions/runs/5545243407/jobs/10123881180) ... and you should be able to use v0.4.0 without any errors (albeit plenty of name change warnings which that migration guide would help with).

I don't _think_ it should cause any ecosystem difficulties if you pin to >0.4.0 ... since v0.4.0 should work for any libraries that haven't updated

Reply:
I basically agree, we might as well go with the latest if there are no obvious problems 🚀

Reply:
happy to help with that.  do you want me to open a PR to remove all the warnings?

Reply:
btw... i'm also working on pydantic2 support (https://github.com/tlambert03/ome-types/pull/205) ... and it's speedy!

<img width="626" alt="Screen Shot 2023-07-13 at 5 50 16 PM" src="https://github.com/bioio-devs/bioio/assets/1609449/e21f5211-790d-4a75-960e-6da2dfc18c92">

(though, I don't think you should pin pydantic yet)

Reply:
@SeanLeRoy should chime in, but I'll just say that one other option is to defer this dependency version update till we move further along on getting bioio ready.   But I'm comfy putting the change both here and aicsimageio anytime

Reply:
yep, sounds good either way.  just ping me if I can be of help

Reply:
I'm willing to try out migrating to `ome-types>=0.4.0`. I'll give it a shot as part of this and update here with my findings though it seems like it should be an easy migration

Reply:
Seemed easy enough 👍 , looks like as part of this `lxml` can be dropped as a direct dependency of this package and installed as an extra from `ome-types` which I did in [#1dcd7ae](https://github.com/bioio-devs/bioio/pull/9/commits/7ac3d9e2939fdcda78a5e18e8cc1d0ad7cdca76f)